vscode-dotnet-runtime-library/src/EventStream/OutputChannelObserver.ts

```diff
@@ -38,7 +38,7 @@ export class OutputChannelObserver implements IEventStreamObserver {
                 }
 
                 const startVersionString = this.inProgressDownloads.join(', ');
-                this.outputChannel.append(`Downloading .NET runtime version(s) ${startVersionString} ...`);
```

Comment:
I think indicating whether we're acquiring the runtime or SDK is still important. Might be worth publishing a separate event (so you don't need to update each of the subevents here) indicating that you're about to acquire an SDK or a runtime etc.